datasafe-test-storages/src/main/java/de/adorsys/datasafe/teststorage/WithStorageProvider.java

```diff
@@ -310,26 +324,32 @@ private static void initS3() {
         }
 
         AmazonS3ClientBuilder amazonS3ClientBuilder = AmazonS3ClientBuilder.standard()
-                .withCredentials(new AWSStaticCredentialsProvider(
-                        new BasicAWSCredentials(amazonAccessKeyID, amazonSecretAccessKey))
-                );
+            .withCredentials(new AWSStaticCredentialsProvider(
+                new BasicAWSCredentials(amazonAccessKeyID, amazonSecretAccessKey))
+            );
 
         if (buckets.size() > 1) {
             log.info("Using {} buckets:{}", buckets.size(), buckets);
         }
 
+
         if (StringUtils.isNullOrEmpty(amazonUrl)) {
-            amazonS3ClientBuilder = amazonS3ClientBuilder.withRegion(amazonRegion);
-            amazonMappedUrl = "s3://" + primaryBucket + "/" + bucketPath + "/";
+            amazonUrl = amazonProtocol + amazonDomain;
+        }
+        final boolean isRealAmazon = ((amazonProtocol + amazonDomain).equals(amazonUrl) || (amazonDomain).equals(amazonUrl));
```

Comment:
Maybe this condition can be simplified to `amazonUrl.endsWtih(amazonDomain)`

Reply:
yes. done